### ISSUE TITLE
Alt-drag does not let me reference blocks

### ISSUE THREAD

**Issue by steveklabnik** [Roam-Research/issues#370]
**Describe the bug**

When I drag, it moves a block, even if I hold down alt.

**To Reproduce**

1. Drag a block to another page while holding down alt
2. The block is moved, not referenced.

**System Information:**
 - Roam version: 0.6.3-f4db89ce
 - Device: Desktop computer
 - OS: Windows 10
 - Browser Chrome

**Comment by roniyaniv** [comment]
Same here.
Mac 10.15.4, Chrome 85.0.4183.83.
Tried incognito mode as well - same result.
Pressing Option key + Dragging simply moves the block from one page to the other.

**Comment by roniyaniv** [comment]
Update - looks like at the moment, you need to press Alt BEFORE starting to drag. 
I would like it to work even if I press Alt AFTER starting to drag.

**Comment by steveklabnik** [comment]
Yes, for me, this is now working. I guess I'll leave it open as a feature request for what @roniyaniv is asking; I would not expect that to work though.

**Comment by mitchschwartz** [comment]
Same issue for the past 6+ months. 
Presently on:
- windows 10 2004 (had it on also on win 19xx) 
- Brave Version 1.13.86 Chromium: 85.0.4183.102 (Official Build) (64-bit) ( with shields off)

**Comment by conwaydean** [comment]
I'm still having this issue on Roam Version 0.7.3-1a824a67
Windows 10 Pro Build 2004 
Microsoft Edge - Version 87.0.634.0 (Official build) dev (64-bit)
